SConstruct

```diff
@@ -20,7 +20,9 @@ if 'SCHROOT_CHROOT_NAME' in os.environ and 'steamrt' in os.environ['SCHROOT_CHRO
 opts = Variables()
 opts.Add(PathVariable("PREFIX", "Directory to install under", "/usr/local", PathVariable.PathIsDirCreate))
 opts.Add(PathVariable("DESTDIR", "Destination root directory", "", PathVariable.PathAccept))
+opts.Add('comp_dir', 'Compilation directory', 'build')
 opts.Add(EnumVariable("mode", "Compilation mode", "release", allowed_values=("release", "debug", "profile")))
+opts.Add('branch', 'Directory inside mode coresponding to current banch. If git is instlled current branch can be found with, git rev-parse --symbolic-full-name --abbrev-ref HEAD', '.')
```

Comment:
There are several typos here: "corresponding," "branch," and "installed." Also, I'm not sure why a separate variable is needed; you could just set `comp-dir` to the branch name or to "build/<branch>" instead.